arwen/elrondapi/elrondei.go

```diff
@@ -572,6 +597,74 @@ func storageLoad(context unsafe.Pointer, keyOffset int32, dataOffset int32) int3
 	return int32(len(data))
 }
 
+//export setStorageLock
+func setStorageLock(context unsafe.Pointer, keyOffset int32, lockTimestamp int64) int32 {
```

Comment:
I chose `int` instead of `unsigned` because `getStorageLock` and `setStorageLock` can also return -1 for errors and didn't want to play to much with casting